public/theme.css

```diff
@@ -116,11 +116,14 @@
 	--theme-bg-accent: hsla(var(--color-purple), var(--theme-accent-opacity));
 	--theme-code-inline-bg: hsla(var(--color-purple), .1);
 	--theme-code-inline-text: var(--theme-text);
-	--theme-code-bg: hsla(257, 31%, 22%, 1);
+	--theme-code-bg: hsla(257, 31%, 25%, 1);
+	--theme-code-tabs: hsla(257, 38%, 35%, 1);
 	--theme-code-text: hsla(var(--color-gray-95), 1);
 	--theme-navbar-bg: var(--theme-bg);
 	--theme-selection-color: hsla(var(--color-purple), 1);
 	--theme-selection-bg: hsla(var(--color-purple), var(--theme-accent-opacity));
+	--theme-code-selection-bg: hsla(var(--color-purple), 0.4);
+	--theme-code-highlight-bg: hsla(var(--color-purple), 0.3);
```

Comment:
```suggestion
	--theme-code-highlight-bg: hsla(var(--color-purple), 0.2);
```

Reply:
Thank you for testing this! I brought the border back as you suggested and fixed background colors until Chrome's built-in color contrast checker was happy. I had to go even further down than 0.2 to make Chrome happy.